src/App.vue

```diff
@@ -160,4 +160,26 @@ export default {
 #app {
 	height: 100%;
 }
+
+// Kludges to ensure Kiva Classic styles are correct when using the .kv-tailwind class
+// Remove once Kiva Classic is the default
+.kv-tailwind {
+	p,
+	dl,
+	ol,
+	ul { line-height: 1.5; }
+}
+
+body .kv-tailwind {
+	font-weight: 300;
```

Comment:
This took me a while to figure out. 
The new prepended-selector in this PR prefixes all tailwind stuff with .kv-tailwind. For the body reset, this looks like
`.kv-tailwind body { font-weight: 300, etc. }`
That selector will never get hit unless you were to put the .kv-tailwind class on the html element. We're not ready to do that yet.  This kludge works around the problem until we can remove the prepended selector.